.github/pull_request_template.md

```diff
@@ -1,26 +1,79 @@
-### This pull request changes...
```

Comment:
This change was merged to the raft repo to update our pull request template. We need to merge it to HHS so it's not included in our other PRs